coordinator/src/main/scala/filodb.coordinator/DatasetCoordinatorActor.scala

```diff
@@ -22,19 +22,21 @@ object DatasetCoordinatorActor {
    * Ingests a bunch of rows into this dataset, version's memtable.
    * Automatically checks at the end if the memtable should be flushed, and sends a StartFlush
    * message to itself to initiate a flush.  This is done before the Ack is sent back.
-   * @param rows the rows to ingest.  Must have the same schema, for now, as the columns passed in
+    *
+    * @param rows the rows to ingest.  Must have the same schema, for now, as the columns passed in
    *        when the ingestion was set up.
    * @return Ack(seqNo)
    */
   case class NewRows(ackTo: ActorRef, rows: Seq[RowReader], seqNo: Long) extends DSCoordinatorMessage
 
   // The default minimum amt of memory in MB to allow ingesting more data
-  val DefaultMinFreeMb = 512
+  val defaultMinFreeMb = 512
```

Comment:
could we keep the uppercase start of the variable name?  That is done intentionally - uppercase variables = constants.